web/style/header.scss

```diff
@@ -0,0 +1,197 @@
+@import "./layout.scss";
+
+.nav-section {
+  display: flex;
+  height: 55px;
+
+  @include break-point(tablet) {
+    height: $header-height;
+  }
+  @include break-point(desktop) {

```

Comment:
This is wrong, see how many media queries are used where as you only need 2 of them, take them below the module and combine the selectors